com.unity.formats.alembic/Editor/Importer/AlembicBuildPostProcess.cs

```diff
@@ -0,0 +1,109 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+using UnityEditor.Callbacks;
+using UnityEngine.Formats.Alembic.Importer;
+using UnityEngine.SceneManagement;
+using Object = UnityEngine.Object;
+
+namespace UnityEditor.Formats.Alembic.Importer
+{
+    public static class AlembicBuildPostProcess
+    {
+        internal static readonly List<KeyValuePair<string,string>> FilesToCopy = new List<KeyValuePair<string,string>>();
+        [PostProcessBuild]
+        public static void OnPostProcessBuild(BuildTarget target, string pathToBuiltProject)
+        {
+            if (target != BuildTarget.StandaloneOSX && target != BuildTarget.StandaloneWindows64)
```

Comment:
you're doing this condition twice, how about factorizing it in a function so if/when we add Linux Standalone there will be less changes to do ?